firestore/src/main/java/com/firebase/ui/firestore/paging/PagingData.java

```diff
@@ -0,0 +1,53 @@
+package com.firebase.ui.firestore.paging;
+
+import android.arch.core.util.Function;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.Transformations;
+import android.arch.paging.PagedList;
+import android.support.annotation.NonNull;
+import android.support.annotation.RestrictTo;
+
+import com.google.firebase.firestore.DocumentSnapshot;
+
+/**
+ * All of the data the adapter needs.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class PagingData {
```

Comment:
I'm wondering if this class is necessary at all... isn't it just casting stuff and forwarding calls along? I feel like it would be simpler to just give our adapter a `PagedList` casted to the Firestore one and get the original live datas. Wha'd ya think?

Reply:
You're absolutely right.  It was only necessary pre beta6

Reply:
Ah, thanks for the explanation, I was just about to comment.